src/components/spinner/spinner.scss

```diff
@@ -0,0 +1,8 @@
+@import '../../../node_modules/spinkit/scss/spinners/9-cube-grid',
```

Comment:
this can be written as `@import '~spinkit/scss/spinners/9-cube-grid'` as web pack resolves ~ to node_modules